### PR TITLE
feat: added commander as color alias.

### PR DESCRIPTION
**Pull request by CVPhelan**

### CODE REVIEW

File: src/parse-query/index.ts [hunk 1]
```diff
@@ -76,6 +76,7 @@ function collectKeywordedQueries(
         parseTags(params, key, operator, value);
         break;
       case "ci":
+      case "commander":
```
Comment (crookedneighbor) [inline src/parse-query/index.ts:79]: Just need to update the test to include this alias: https://github.com/Commander-Spellbook/commander-spellbook-api-wrapper/blob/9e219247071a08fdc4adaf989b7ba40cb81c5e12/test/unit/parse-query/index.test.ts#L570

**Review by crookedneighbor — CHANGES_REQUESTED** [review]

File: src/parse-query/index.ts (hunk 1 above)
Comment (crookedneighbor) [reply src/parse-query/index.ts:79]: Also need to add `-commander` for the negative case.

**Review by crookedneighbor — CHANGES_REQUESTED** [review]
Need tests: https://github.com/Commander-Spellbook/commander-spellbook-api-wrapper/blob/9e219247071a08fdc4adaf989b7ba40cb81c5e12/test/unit/parse-query/index.test.ts#L542-L568

**Comment by CVPhelan** [comment]
> Need tests:
> 
> https://github.com/Commander-Spellbook/commander-spellbook-api-wrapper/blob/9e219247071a08fdc4adaf989b7ba40cb81c5e12/test/unit/parse-query/index.test.ts#L542-L568

need clarification here.

**Pull request merged**